src/slurm_plugin/slurm_resources.py

```diff
@@ -246,6 +254,14 @@ def get_compute_resource_name(self):
         _, _, compute_resource_name = parse_nodename(self.name)
         return compute_resource_name
 
+    def _parse_error_code(self):
+        """Parse RunInstance error code from node reason."""
+        if self.reason and self.reason.startswith("(Code:"):
+            index_of_bracket = self.reason.find(")")
+            error_code = self.reason[6:index_of_bracket]  # 6 is the length of "(Code"
```

Comment:
nit: what if we replace 6 with len("(Code") so that we have self documenting code?

Reply:
Replaced